FeedStoreChallenge/MemoryFeedStore.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  MemoryFeedStore.swift
+//  FeedStoreChallenge
+//
+//  Created by Damian Modernell on 18/3/2564 BE.
+//  Copyright © 2564 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+public final class MemoryFeedStore: FeedStore {
+	
+	private var feedCache: ([LocalFeedImage], Date) = ([], Date())
```

Comment:
The default values are not needed - you can use an optional to represent the empty cache:

```suggestion
	private var feedCache: ([LocalFeedImage], Date)?
```

Reply:
👍